sysutils/smart/src/opnsense/service/conf/actions.d/actions_smart.conf

```diff
@@ -57,27 +57,32 @@ command:/usr/local/sbin/smartctl -t offline
 parameters:%s; exit 0
 type:script_output
 message:Testing device %s (offline)
+description:Run SMART test (offline)
 
 [test.short]
 command:/usr/local/sbin/smartctl -t short
 parameters:%s; exit 0
 type:script_output
 message:Testing device %s (short)
+description:Run SMART test (short)
 
 [test.long]
 command:/usr/local/sbin/smartctl -t long
 parameters:%s; exit 0
 type:script_output
 message:Testing device %s (long)
+description:Run SMART test (long)
 
 [test.conveyance]
 command:/usr/local/sbin/smartctl -t conveyance
 parameters:%s; exit 0
 type:script_output
 message:Testing device %s (conveyance)
+description:Run SMART test (conveyance)
 
 [abort]
 command:/usr/local/sbin/smartctl -X
 parameters:%s; exit 0
 type:script_output
 message:Abort test on device %s
+description:Abort SMART tests
```

Comment:
I'm unsure this makes sense in cron environments, but otherwise ok

Reply:
There are times when SMART tests get stuck. Maybe someone would like to run cleanup once a month or so just to be sure or just abort any long-running test before starting a new one.